projects/plugins/boost/app/modules/optimizations/minify/class-js.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Automattic\Jetpack_Boost\Modules\Optimizations\Minify;
+
+use Automattic\Jetpack_Boost\Contracts\Pluggable;
+
+class Js extends Minify implements Pluggable {
+
+	public static $default_excludes = array( 'jquery', 'jquery-core', 'underscore', 'backbone' );
+
+	public static function get_slug() {
+		return 'minify_js';
+	}
+
+	public function init_concatenate() {
+		if ( is_admin() ) {
+			return;
+		}
+
+		jetpack_boost_init_filesystem();
+
+		global $wp_scripts;
+
+		// phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
+		$wp_scripts                         = new Concatenate_JS( $wp_scripts );
+		$wp_scripts->allow_gzip_compression = true; // @todo - used constant ALLOW_GZIP_COMPRESSION = true if not defined.
```

Comment:
TODOs should be turned into tickets.

Reply:
Opened an issue listing all todos https://github.com/Automattic/jetpack/issues/30080.